ansible/roles/de-TMP-RUN-ONCE-app-beta-tagger/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- debug:
+    msg: "This role should only be RUN ONCE against the metadata database. It will fail if run more than once."
+
+- name: update docker-py
+  pip: name="docker-py" state="latest"
+
+- name: pull the apps beta tagger docker image
+  docker_image: name="{{ app_beta_tagger_repo }}" tag="{{ docker_tag }}"
+
+- name: tag DE beta apps
+  shell: "docker run --rm
```

Comment:
I resorted to a `shell` command here because the `docker_container` version of this command was not running the conversion for some reason, but still returning success to the task.

I was also not sure how to capture the `docker_container` command's output to debug the issue.